geoopt/manifolds/stereographic/manifold.py

```diff
@@ -418,6 +449,53 @@ def random_normal(
 
     random = random_normal
 
+    def wrapped_normal(
+        self, mean: torch.Tensor, std, size, dtype=None, device=None
```

Comment:
Make API similar to `normal`
```suggestion
        self, *size, mean: torch.Tensor, std=1, dtype=None, device=None
```